lib/pf/util.pm

```diff
@@ -502,6 +502,16 @@ sub fix_file_permissions {
     pf_run('sudo /usr/local/pf/bin/pfcmd fixpermissions file "' . $file . '"');
 }
 
+=item fix_files_permissions
```

Comment:
Maybe a better name fix_permissions_all.
I find the name to close to the other function.